notebooks/en/semantic_reranking_elasticsearch.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>Would also be nice to quickly mention what Eland is. Could be something like:</p><p> </p><p>"Upload the model to your Elasticsearch deployment with Eland, a Python client for machine learning"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>